src/core/core.h

```diff
@@ -54,6 +39,8 @@ __extension__({				\
 })
 
 #ifndef CACHE_LINE_SIZE // TODO: calculate and inject at build time
+/// The size of a cpu cache line
```

Comment:
Maybe this is breaking Doxygen syntax and only the second comment is being included in the doc.

Reply:
I answered to your other similar remark